clients/drcachesim/CMakeLists.txt

```diff
@@ -965,6 +965,23 @@ if (BUILD_TESTS)
       use_DynamoRIO_extension(tool.drcacheoff.burst_traceopts drcovlib_static)
     endif (NOT RISCV64)
 
+    if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+      set(source_abs "${CMAKE_CURRENT_SOURCE_DIR}/tests/window_test.cpp")
+      add_executable(tool.drcacheoff.windows-timestamps ${source_abs})
+      configure_DynamoRIO_static(tool.drcacheoff.windows-timestamps)
+      use_DynamoRIO_static_client(tool.drcacheoff.windows-timestamps drmemtrace_static)
+      target_link_libraries(tool.drcacheoff.windows-timestamps drmemtrace_raw2trace
+        drmemtrace_analyzer test_helpers)
+      if (WIN32)
+        # Just like for burst_replace, linking everything together takes effort.
+        target_link_libraries(tool.drcacheoff.windows-timestamps ${static_libc})
+      endif ()
+      set(tool.drcacheoff.windows-timestamps_uses_configure ON) # Avoid dup DEBUG.
+      add_win32_flags(tool.drcacheoff.windows-timestamps)
+      use_DynamoRIO_drmemtrace_tracer(tool.drcacheoff.windows-timestamps)
+      use_DynamoRIO_extension(tool.drcacheoff.windows-timestamps drcovlib_static)
```

Comment:
Does it actually use drcovlib?  I don't see any use.

Reply:
Ack. Removed it.